.github/workflows/release-spark.yml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Databricks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Release Spark
```

Comment:
Upon request (manual trigger), we can deploy a new version of our generated jar file to maven central (under FINOS) so that anyone can use this LCR package as-is on spark environment. Note: this requires adding credentials as secret (Finos team)